<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerClass</class>
 <widget class="QMainWindow" name="ManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>703</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>703</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>703</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1250</width>
    <height>703</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page1">
       <property name="styleSheet">
        <string notr="true">#page1{border-image: url(:/Manager/Resources/login.jpg);}
</string>
       </property>
       <widget class="QLineEdit" name="lineEditUserName">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>280</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="placeholderText">
         <string>请输入用户名</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditPassWord">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>350</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑體 Bold</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnLogIn">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>430</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#page2{border-image: url(:/Manager/Resources/workbg.jpg);}</string>
       </property>
       <widget class="QComboBox" name="comboBoxReportType">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>日报</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>周报</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>月报</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>年报</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="labelReportType">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelReportType{background-color:rgba(255, 255, 255, 60%);}</string>
        </property>
        <property name="text">
         <string>请选择统计报表类型</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="stackedWidgetTimeSelection">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="pageDay">
         <widget class="QLabel" name="labelDay">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>241</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>思源黑体 Bold</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#labelDay{background-color:rgba(255, 255, 255, 60%);}</string>
          </property>
          <property name="text">
           <string>请选择日报日期</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pageWeek">
         <widget class="QLabel" name="labelWeek">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>241</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>思源黑体 Bold</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#labelWeek{background-color:rgba(255, 255, 255, 60%);}</string>
          </property>
          <property name="text">
           <string>请选择周报所在周任意日期</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pageMonth">
         <widget class="QLabel" name="labelMonth">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>241</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>思源黑体 Bold</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#labelMonth{background-color:rgba(255, 255, 255, 60%);}</string>
          </property>
          <property name="text">
           <string>请选择月报所在月任意日期</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pageYear">
         <widget class="QLabel" name="labelYear">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>241</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>思源黑体 Bold</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#labelYear{background-color:rgba(255, 255, 255, 60%);}</string>
          </property>
          <property name="text">
           <string>请选择年报所在年任意日期</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>241</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnReportQuery">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>530</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>查阅报表</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableReport">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>941</width>
          <height>621</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#tableReport{background-color:rgba(255, 255, 255, 60%);}</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelShowReport">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>941</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelShowReport{background-color:rgba(255, 255, 255, 60%);}</string>
        </property>
        <property name="text">
         <string>请查阅报表：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnPrint">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>590</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>导出报表</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxRoomID">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>470</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="labelRoomID">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>思源黑体 Bold</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelRoomID{background-color:rgba(255, 255, 255, 60%);}</string>
        </property>
        <property name="text">
         <string>请选择您要查看的房间ID</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Manager.qrc"/>
 </resources>
 <connections/>
</ui>
